### DISCUSSION TITLE
Can anyone share me how to install Ruby in VS Code (Windows) step by step and also the steps to run and debug using the Jsonfile.

### DISCUSSION THREAD

**Discussion by JIPrashandh** [community/community#30796] [Codespaces]
Can anyone share me how to install Ruby in VS Code (Windows) step by step and also the steps to run and debug using the Jsonfile.

Installation with ruby extensions like ruby and ruby debug is done and after that how can use run and debug for the output we are program that we are trying.

Thanks awaiting reply.

**Comment by jfullstackdev** [comment]
maybe this one 

https://www.geeksforgeeks.org/how-to-configure-vscode-for-ruby-developers/

**Reply by JIPrashandh** [reply]
@jdevstatic or anyone. I Followed it before and I can run a ruby code But I want to see the variables,watch, call stack and breakpoints in the run and debug. When I enter the ruby code and then I click run without debugging and in the run and debug tab it runs and disappears. So I want to see the variables call stack breakpoints here.
![image](https://user-images.githubusercontent.com/106915201/186835442-3aa21b65-0c27-4320-8897-637e42ef3d2f.png)
But I believe need to configure the json file or I am not sure.

**Reply by JIPrashandh** [reply]
When I click start debugging I can see 
![image](https://user-images.githubusercontent.com/106915201/186839360-c517dd47-5ed8-4aed-85f4-d6b53ecd0b5e.png)
So what is the configuration file for Ruby...we need to add and how. What is the syntax for the file.
